cypress/integration/app/settingsflow.spec.ts

```diff
@@ -137,4 +137,17 @@ describe("Settings flow", () => {
 
     cy.findByLabelText(/host expiry window/i).should("have.value", "5");
   });
+
+  it("if last email is Hello from Fleet", () => {
+    cy.request("http://localhost:8025/api/v2/messages").then((response) => {
```

Comment:
Can you move this into a "command" in the `commands.ts` file? I think it will be easier to understand this and more clear to future test-writers how to use it as `cy.getEmails().then...`